src/test/java/bio/terra/service/journal/JournalServiceTest.java

```diff
@@ -82,21 +85,20 @@ public void journalCreateTest_SameKeyDifferentResourceType() {
   @Test
   public void journalGetResults_InvalidOffset() {
     UUID key = UUID.randomUUID();
-    assertThat(
+    DataIntegrityViolationException ex =
```

Comment:
Oh man, this is the thing that kills me about the hamcrest matchers.  Having to do stuff like this for error checks.  Ah well

Reply:
Yeah…. I found this slightly more readable than the existing, which nested assertions.  I can use a `var ex = …` if you'd prefer!